database/factories/CommentFactory.php

```diff
@@ -0,0 +1,32 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Database\Factories;
+
+use App\Models\Comment;
+use Illuminate\Database\Eloquent\Factories\Factory;
+
+/**
+ * @extends Factory<Comment>
+ */
+class CommentFactory extends Factory
+{
+    protected $model = Comment::class;
+
+    public function definition(): array
+    {
+        return [
+            'Payload' => $this->faker->paragraph,
+            'Submitted' => $this->faker->dateTimeBetween('-1 year', 'now'),
+            'Edited' => $this->faker->dateTimeBetween('now', '+1 year'),
+            'user_id' => 1,
+            'commentable_id' => $this->faker->numberBetween(1, 100),
+            'commentable_type' => $this->faker->randomElement([
+                'App\Models\Article',
+                'App\Models\Post',
+                'App\Models\Product',
+            ]),
```

Comment:
Review pending, but I need this factory for something else so leaving a comment here 😅

Let's omit these two from the factory for now as they're currently unused in production.

Reply:
This was generated with `php artisan make:factory CommentFactory` iirc. I'll remove the commentables and add the Articles.